CustomCommands/lua/autorun/client/cl_dban.lua

```diff
@@ -123,99 +122,76 @@ function OpenPanelZZ4Z( ply, cmd, args, str )
 					RunConsoleCommand( "ulx", "banip", Time:GetText(), ( list:GetLine( line ):GetValue( 3 ) ) )
 					Frame:Close()
 				end
-				
+
 				local cancelbutton = vgui.Create( "DButton", Frame )
 				cancelbutton:SetSize( 75, 20 )
 				cancelbutton:SetPos( 127, 73 )
 				cancelbutton:SetText( "Cancel" )
 				cancelbutton.DoClick = function( cancelbutton )
 					Frame:Close()
 				end
-				
+
 			end ):SetIcon("icon16/vcard_delete.png")
-			
+
         menu:AddOption( "Copy Name", function()
 			SetClipboardText( tostring( list:GetLine( line ):GetValue( 1 ) ) )
 		end ):SetIcon("icon16/user_edit.png")
-		
+
 		menu:AddOption( "Copy SteamID", function()
 			SetClipboardText( tostring( list:GetLine( line ):GetValue( 2 ) ) )
 		end ):SetIcon("icon16/tag_blue_edit.png")
-		
+
 		if ply:IsAdmin() then
 			menu:AddOption( "Copy IP Address", function()
 				SetClipboardText( tostring( list:GetLine( line ):GetValue( 3 ) ) )
 			end ):SetIcon("icon16/vcard_edit.png")
 		end
-		
+
 		menu:AddOption( "Copy Time", function()
 			SetClipboardText( tostring( list:GetLine( line ):GetValue( 4 ) ) )
 		end ):SetIcon("icon16/time.png")
-		
+
 		menu:AddOption( "View Profile", function()
 			gui.OpenURL("http://steamcommunity.com/profiles/".. util.SteamIDTo64( tostring( list:GetLine( line ):GetValue( 2 ) ) ) )
 		end ):SetIcon("icon16/world.png")
-		
+
 		if ply:IsAdmin() then
 			menu:AddOption( "Whois", function()
 				gui.OpenURL("http://whois.net/ip-address-lookup/".. tostring( list:GetLine( line ):GetValue( 3 ) ) )
 			end ):SetIcon("icon16/zoom.png")
 		end
-		
+
 		menu:Open()
-		
-	end
-	
-	if ( table.Count( disconnectTable ) == 0 ) then
-		RunConsoleCommand( "discs_request" )
-		
-		timer.Simple( 1, function()
-			for k,v in pairs( disconnectTable ) do
-				if ply:IsAdmin() then
-					list:AddLine( v[1], k, v[2], v[3] )
-				else
-					local x = string.gsub( v[2], "%d", "-" )
-					list:AddLine( v[1], k, x, v[3] )
-				end
-			end
-			list:SortByColumn( 4, true )
-			main:ShowCloseButton( true )
-		end )
 
-		return
-		
 	end
 
-	for k,v in pairs( disconnectTable ) do
+	RunConsoleCommand( "discs_request" )
+
+	recv = function()
+		if not IsValid(main) then return end
 		if ply:IsAdmin() then
-			list:AddLine( v[1], k, v[2], v[3] )
+			for k, v in ipairs(disconnectTable) do
+				list:AddLine(v[2], v[1], v[3], v[4])
+			end
 		else
-			local x = string.gsub( v[2], "%d", "-" )
-			list:AddLine( v[1], k, x, v[3] )
+			for k,v in ipairs( disconnectTable ) do
+				local x =
```

Comment:
accidentally left this line in?

Reply:
Fixed. Slipped past me because it didn't end up causing any problems and the linter didn't pick it up.